lib/trento/settings.ex

```diff
@@ -154,6 +155,44 @@ defmodule Trento.Settings do
     end
   end
 
+  # Analytics settings
+
+  @spec get_analytics_settings() ::
+          {:ok, AnalyticsSettings.t()} | {:error, :analytics_settings_not_configured}
+  def get_analytics_settings do
+    case Repo.one(AnalyticsSettings.base_query()) do
+      %AnalyticsSettings{} = settings -> {:ok, settings}
+      nil -> {:error, :analytics_settings_not_configured}
+    end
+  end
+
+  @spec change_analytics_opt_in(boolean()) ::
+          {:ok, AnalyticsSettings.t()}
+          | {:error, :analytics_settings_not_configured}
+  def change_analytics_opt_in(value) do
+    case Repo.one(AnalyticsSettings.base_query()) do
+      %AnalyticsSettings{} = settings ->
+        {:ok, settings}
```

Comment:
You don't need this `{:ok, settings}`. You can simply remove it.

Reply:
This has been removed.